encoding/ascii.go

```diff
@@ -6,7 +6,10 @@ import (
 	"github.com/moov-io/iso8583/utils"
 )
 
-var ASCII Encoder = &asciiEncoder{}
```

Comment:
It's not convenient to have ASCII (and the rest) as Encoder - you can't navigate easily, etc. They can be their own types.